website/source/docs/secrets/gcp/index.html.md

```diff
@@ -29,6 +29,19 @@ The benefits of using this secrets engine to manage Google Cloud IAM service acc
   using a central identity service (such as LDAP) and generate GCP credentials
   without the need to create or manage a new Service Account for that user.
 
+
+~> **WARNING (Deprecation of `access_token` Leases)**: In previous versions of this
```

Comment:
Let's be more specific here. What versions? When is it no longer an issue? How can people check if they are vulnerable?

Reply:
I guess... I'm not actually in favor of adding this to the main docs. I think this is better-suited for the changelog. I'm thinking of a new user coming to the docs and seeing this huge warning. It feels unwelcoming.